README.md

```diff
@@ -0,0 +1,94 @@
+# gr-stellarstation: GNURadio blocks for interacting with StellarStation
+
+gr-stellarstation is an out-of-tree GNURadio module that provides blocks for easily integrating with the [StellarStation](https://www.stellarstation.com/) API in GNURadio flowgraphs.
+
+## Installation
+
+### PyBOMBS
+
+PyBOMBs support is currently under construction.
+
+### Source Installation
+
+gr-stellarstation, like most OOT modules, can be built from source using CMake.
+
+First, we need to install gr-stellarstation's sole dependency, [Conan](https://conan.io/). The simplest way to do this is through `pip`:
+
+```bash
+$ pip install conan
+```
+
+For alternative methods to install Conan, see the [Conan documentation](https://docs.conan.io/en/latest/installation.html).
+
+After installing Conan, run the following commands to add the relevant Conan recipes:
+
+```bash
+$ conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+$ conan remote add inexorgame https://api.bintray.com/conan/inexorgame/inexor-conan
+$ conan remote add stellarstation https://api.bintray.com/conan/infostellarinc/stellarstation-conan
+```
+
+Proceed to install gr-stellarstation normally with CMake:
+
+```bash
+$ git clone https://github.com/infostellarinc/gr-stellarstation.git
+$ cd gr-stellarstation
+$ mkdir build
+$ cmake ..
+$ make
+$ sudo make install
+$ sudo ldconfig
+```
+
+## Examples
+
+See the [examples directory](examples/) for flowgraphs showing the usage of gr-stellarstation blocks.
+
+## StellarStation API Key
+
+To use most of the blocks in this module, you will need a StellarStation API Key, which is a .json file containing credentials to access the StellarStation API. An API key can be generated in the StellarStation Console.
+
+## Quickstart
+
+A typical use case of gr-stellarstation is direct streaming of complex I/Q data from a satellite tracked by StellarStation to a GNURadio flowgraph in real-time.
+
+gr-stellarstation provides the block **StellarStation IQ Source** for this purpose.
+
+![iq_source_sample](docs/images/iq_source_sample.png)
+
+Note that we've configured the block to connect to *Satellite ID* "73" (corresponding to the weather satellite [NOAA 19](https://en.wikipedia.org/wiki/NOAA-19)) and given it the path of the StellarStation API Key via *StellarStation API Key Filepath*.
```

Comment:
Btw, I think retrieviing the ID of a particular satellite is not that obvious. Is there a suggested way to do this for external users?

Reply:
The ID will be available in the satellite console.